### PR TITLE
Elevator roller states -> subsystems-dev

### PR DESCRIPTION
**Pull request by Yotam7**

### CODE REVIEW

File: src/main/java/frc/robot/subsystems/elevatorRoller/ElevatorRollerState.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
+package frc.robot.subsystems.elevatorRoller;
+
+public enum ElevatorRollerState {
```
Comment (YHerm) [inline src/main/java/frc/robot/subsystems/elevatorRoller/ElevatorRollerState.java:3]: u probably will want one that goes reverse

File: src/main/java/frc/robot/subsystems/elevatorRoller/ElevatorRollerState.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+	INTAKE_TO_ELEVATOR_ROLLER(0),
```
Comment (YHerm) [inline src/main/java/frc/robot/subsystems/elevatorRoller/ElevatorRollerState.java]:
```suggestion
	TRANSFER_TO_ELEVATOR(0),
```

**Review by YHerm — CHANGES_REQUESTED** [review]
uupdtae with subsystems-dev

**Review by drorta — DISMISSED** [review]

**Review by maya1414 — DISMISSED** [review]

File: src/main/java/frc/robot/superstructure/Superstructure.java [hunk 3]
```diff
@@ -56,71 +64,81 @@ public Command setState(RobotState state) {
...
 				.until(() -> swerve.isAtHeading(Field.getAngleToAmp())),
+			elevatorRollerStateHandler.setState(ElevatorRollerState.AMP),
 			funnelStateHandler.setState(FunnelState.AMP)
 		);
 	}
 
 	public Command transferShooterElevator() {
 		return new ParallelCommandGroup(
 			swerve.getCommandsBuilder().saveState(SwerveState.DEFAULT_DRIVE),
+			elevatorRollerStateHandler.setState(ElevatorRollerState.STOP),
```
Comment (greenblitz4590) [inline src/main/java/frc/robot/superstructure/Superstructure.java]: to elevatro

File: src/main/java/frc/robot/superstructure/Superstructure.java [hunk 4]
```diff
@@ -56,71 +64,81 @@ public Command setState(RobotState state) {
...
 			funnelStateHandler.setState(FunnelState.SHOOTER_TO_ELEVATOR)
 		);
 	}
 
 	public Command transferElevatorShooter() {
 		return new ParallelCommandGroup(
 			swerve.getCommandsBuilder().saveState(SwerveState.DEFAULT_DRIVE),
+			elevatorRollerStateHandler.setState(ElevatorRollerState.STOP),
```
Comment (greenblitz4590) [inline src/main/java/frc/robot/superstructure/Superstructure.java]: from elevator

File: src/main/java/frc/robot/superstructure/Superstructure.java [hunk 5]
```diff
@@ -56,71 +64,81 @@ public Command setState(RobotState state) {
...
 			funnelStateHandler.setState(FunnelState.NOTE_TO_SHOOTER)
 		);
 	}
 
 	public Command intakeOuttake() {
 		return new ParallelCommandGroup(
 			swerve.getCommandsBuilder().saveState(SwerveState.DEFAULT_DRIVE),
+			elevatorRollerStateHandler.setState(ElevatorRollerState.STOP),
```
Comment (greenblitz4590) [inline src/main/java/frc/robot/superstructure/Superstructure.java]: reverse, or from elevator

**Review by maya1414 — DISMISSED** [review]

File: src/main/java/frc/robot/superstructure/Superstructure.java [hunk 6]
```diff
@@ -56,71 +64,81 @@ public Command setState(RobotState state) {
...
 
 	public Command speaker() {
 		return new ParallelCommandGroup(
-			funnelStateHandler.setState(FunnelState.SPEAKER),
-			swerve.getCommandsBuilder().saveState(SwerveState.DEFAULT_DRIVE.withAimAssist(AimAssist.SPEAKER))
-		);
+			swerve.getCommandsBuilder().saveState(SwerveState.DEFAULT_DRIVE.withAimAssist(AimAssist.SPEAKER)),
+			elevatorRollerStateHandler.setState(ElevatorRollerState.STOP),
+			funnelStateHandler.setState(FunnelState.SPEAKER)
+			);
```
Comment (YHerm) [inline src/main/java/frc/robot/superstructure/Superstructure.java]: tab less

**Review by maya1414 — DISMISSED** [review]

**Review by drorta — DISMISSED** [review]

**Review by hillelv — DISMISSED** [review]

**Review by YHerm — DISMISSED** [review]

File: src/main/java/frc/robot/subsystems/elevatorRoller/calibarations [hunk 7]
```diff
@@ -2,4 +2,5 @@ Calibrations:
 
 1. Motor id
 2. Current limit
-3. Gear ratio
\ No newline at end of file
+3. Gear ratio
+4. What power should be for INTAKE_TO_ELEVATOR_ROLLER and AMP
```
Comment (bossyshrimp68) [inline src/main/java/frc/robot/subsystems/elevatorRoller/calibarations]: "Power for ..." will do

**Review by bossyshrimp68 — DISMISSED** [review]

**Review by maya1414 — APPROVED** [review]

**Pull request merged**